src/test/java/com/flightstats/hub/model/ChannelConfigTest.java

```diff
@@ -189,4 +185,34 @@ public void testZeroBytes() {
         assertFalse(testZeroBytes.isAllowZeroBytes());
     }
 
+    @Test
+    public void testSecondaryMetricsReporting_defaultEmpty() {
+        ChannelConfig testSecondaryMetricsNull = ChannelConfig
+                .builder()
+                .name("testSecondaryMetrics0")
+                .build();
+        assertEquals("", testSecondaryMetricsNull.getSecondaryMetricsReporting());
```

Comment:
If you change this to a lombok object, you'll either need to add a default value on this or write your own getter. I guess either one isn't too bad...

Reply:
On the bright side, if you just do a default value, you can probably get rid of the next test down.